easy_infra.yml

```diff
@@ -108,21 +108,21 @@ packages:
       env_vars:
       - TF_DATA_DIR
     security: *id004
-    version: v1.6.0-rc1
+    version: v1.7.3
```

Comment:
This is the result of the fix; when we initially introduced opentofu they didn't have any GA releases yet so we couldn't include it in the auto updates.